app/models/task.rb

```diff
@@ -15,10 +15,11 @@ class Task < ActiveRecord::Base
   scope :not_started, -> { where(status: NOT_STARTED) }
 
   def start!
-    update_attribute(:status, STARTED)
+    update_attributes(status: STARTED,started_at: Time.now.strftime('%Y-%m-%d %H:%M'))
```

Comment:
`update_attributes(status: STARTED,started_at: Time.now)`

`Time.now` returns correct object that can be set to the started_at field